{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "5d2e8b5a-a716-4866-8e73-7c03127e98af",
   "metadata": {},
   "source": [
    "![Notebook Banner](cosmos-predict1_banner.png)\n",
    "\n",
    "## Set Up Docker Login (Executing on A Terminal Locally)\n",
    "\n",
    "Replace <your api key> with your own NGC API Key\n",
    "\n",
    "```bash\n",
    "export NGC_CLI_API_KEY=\"nvapi-xxx\"\n",
    "echo \"$NGC_CLI_API_KEY\" | docker login nvcr.io --username '$oauthtoken' --password-stdin\n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5cbc68f3-4d9a-4322-ac35-7599eeb981e7",
   "metadata": {},
   "source": [
    "## Deploy NIM (Executing on this Jupyter Notebook)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "802fad26",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "37e4944c0567b7f6057dff2e67f8ac22e5e6d8abe7520808b3221b18f36eb595\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "export NGC_API_KEY=\"nvapi-xxx\"\n",
    "\n",
    "export CONTAINER_NAME=\"cosmos-predict1-7b-text2world\"\n",
    "export IMG_NAME=\"nvcr.io/nim/nvidia/cosmos-predict1-7b-text2world:1.0.0\"\n",
    "\n",
    "# A path on your system to cache the downloaded models\n",
    "export LOCAL_NIM_CACHE=/ephemeral/.cache/nim\n",
    "mkdir -p $LOCAL_NIM_CACHE\n",
    "\n",
    "# Start the NVIDIA NIM for Cosmos\n",
    "docker run -d --rm --name=$CONTAINER_NAME \\\n",
    "    --runtime=nvidia \\\n",
    "    --gpus all \\\n",
    "    --shm-size=16GB \\\n",
    "    -e NGC_API_KEY \\\n",
    "    -v $LOCAL_NIM_CACHE:/opt/nim/.cache \\\n",
    "    -u $(id -u) \\\n",
    "    -p 8000:8000 \\\n",
    "    $IMG_NAME"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "77b41343",
   "metadata": {},
   "source": [
    "### Make Sure the Contianer is Up and Running\n",
    "The following message indicates a successful startup.\n",
    "\n",
    "{'message': 'Starting HTTP Inference server', 'port': 8000, 'workers_count': 1, 'host': '0.0.0.0', 'log_level': 'info', 'SSL': 'disabled'}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "97dbaf3b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=============================\n",
      "== Triton Inference Server ==\n",
      "=============================\n",
      "\n",
      "NVIDIA Release 25.01 (build 136230209)\n",
      "Triton Server Version 2.54.0\n",
      "\n",
      "Copyright (c) 2018-2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.\n",
      "\n",
      "Various files include modifications (c) NVIDIA CORPORATION & AFFILIATES.  All rights reserved.\n",
      "\n",
      "This container image and its contents are governed by the NVIDIA Deep Learning Container License.\n",
      "By pulling and using the container, you accept the terms and conditions of this license:\n",
      "https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license\n",
      "\n",
      "NOTE: CUDA Forward Compatibility mode ENABLED.\n",
      "  Using CUDA 12.8 driver version 570.86.10 with kernel driver version 535.183.06.\n",
      "  See https://docs.nvidia.com/deploy/cuda-compatibility/ for details.\n",
      "\n",
      "INFO 2025-07-30 11:56:07.598] Starting nimlib 0.8.3 nim_sdk 0.7.6\n",
      "INFO 2025-07-30 11:56:07.598] NIM VERSION:\n",
      "1.0.0\n",
      "INFO 2025-07-30 11:56:07.598] NIM NOTICE:\n",
      "GOVERNING TERMS: The NIM container is governed by the NVIDIA Software License Agreement and Product-Specific Terms for AI Products; and the use of this model is governed by the NVIDIA Open Model License.\n",
      "A copy of theses licenses can be found under /opt/nim/LICENSE.\n",
      "\n",
      "https://www.nvidia.com/en-us/agreements/enterprise-software/nvidia-software-license-agreement/\n",
      "https://www.nvidia.com/en-us/agreements/enterprise-software/product-specific-terms-for-ai-products/\n",
      "https://www.nvidia.com/en-us/agreements/enterprise-software/nvidia-open-model-license/\n",
      "INFO 2025-07-30 11:56:07.598] NIM banner.txt:\n",
      "The cosmos-predict1-7b-text2world model is licensed under the NVIDIA Open Model License Agreement.\n",
      "\n",
      "https://www.nvidia.com/en-us/agreements/enterprise-software/nvidia-open-model-license/\n",
      "\n",
      "INFO 2025-07-30 11:56:11.728] Registered custom profile selectors: [<class 'inference.CosmosProfileSelector'>]\n",
      "INFO 2025-07-30 11:56:11.813] Matched profile_id in manifest from CosmosProfileSelector ad9e1b42331f5453f5b041830cebc89ea65c6fffacdc22962ecb4be3173559f6 with tags: {'llm_precision': 'int8', 'precision': 'bf16', 'profile': 'latency', 'required_ram': '91268055040'}\n",
      "INFO 2025-07-30 11:56:11.813] Using the profile selected by the profile selector: ad9e1b42331f5453f5b041830cebc89ea65c6fffacdc22962ecb4be3173559f6\n",
      "INFO 2025-07-30 11:56:11.814] Downloading manifest profile: ad9e1b42331f5453f5b041830cebc89ea65c6fffacdc22962ecb4be3173559f6\n",
      "INFO 2025-07-30 11:56:11.817 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.825 lib.rs:204] File: tokenizer_config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.825 public.rs:52] Skipping download, using cached copy of file: tokenizer_config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.826 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.831 lib.rs:204] File: spiece.model found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/spiece.model\"\n",
      "INFO 2025-07-30 11:56:11.831 public.rs:52] Skipping download, using cached copy of file: spiece.model at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/spiece.model\"\n",
      "INFO 2025-07-30 11:56:11.832 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.838 lib.rs:204] File: model.onnx found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-onnx-bf16-2/model.onnx\"\n",
      "INFO 2025-07-30 11:56:11.838 public.rs:52] Skipping download, using cached copy of file: model.onnx at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-onnx-bf16-2/model.onnx\"\n",
      "INFO 2025-07-30 11:56:11.839 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.845 lib.rs:204] File: special_tokens_map.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.845 public.rs:52] Skipping download, using cached copy of file: special_tokens_map.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.846 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.852 lib.rs:204] File: tokenizer_config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.852 public.rs:52] Skipping download, using cached copy of file: tokenizer_config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.852 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.858 lib.rs:204] File: autoencoder.jit found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/autoencoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.858 public.rs:52] Skipping download, using cached copy of file: autoencoder.jit at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/autoencoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.859 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.865 lib.rs:204] File: mean_std.pt found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/mean_std.pt\"\n",
      "INFO 2025-07-30 11:56:11.865 public.rs:52] Skipping download, using cached copy of file: mean_std.pt at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/mean_std.pt\"\n",
      "INFO 2025-07-30 11:56:11.866 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.871 lib.rs:204] File: decoder.jit found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/decoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.871 public.rs:52] Skipping download, using cached copy of file: decoder.jit at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/decoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.872 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.878 lib.rs:204] File: blocklist.tar found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-blocklist/blocklist.tar\"\n",
      "INFO 2025-07-30 11:56:11.878 public.rs:52] Skipping download, using cached copy of file: blocklist.tar at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-blocklist/blocklist.tar\"\n",
      "INFO 2025-07-30 11:56:11.879 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.885 lib.rs:204] File: rank0.safetensors found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/rank0.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.885 public.rs:52] Skipping download, using cached copy of file: rank0.safetensors at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/rank0.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.886 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.892 lib.rs:204] File: tokenizer.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.892 public.rs:52] Skipping download, using cached copy of file: tokenizer.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.893 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.898 lib.rs:204] File: spiece.model found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-tokenizer/spiece.model\"\n",
      "INFO 2025-07-30 11:56:11.898 public.rs:52] Skipping download, using cached copy of file: spiece.model at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-tokenizer/spiece.model\"\n",
      "INFO 2025-07-30 11:56:11.899 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.905 lib.rs:204] File: config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/config.json\"\n",
      "INFO 2025-07-30 11:56:11.905 public.rs:52] Skipping download, using cached copy of file: config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/config.json\"\n",
      "INFO 2025-07-30 11:56:11.907 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.912 lib.rs:204] File: nemo.tar found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-nemo/nemo.tar\"\n",
      "INFO 2025-07-30 11:56:11.912 public.rs:52] Skipping download, using cached copy of file: nemo.tar at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-nemo/nemo.tar\"\n",
      "INFO 2025-07-30 11:56:11.913 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.919 lib.rs:204] File: encoder.jit found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/encoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.919 public.rs:52] Skipping download, using cached copy of file: encoder.jit at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/encoder.jit\"\n",
      "INFO 2025-07-30 11:56:11.920 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.926 lib.rs:204] File: model.onnx found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-detect-faces-onnx/model.onnx\"\n",
      "INFO 2025-07-30 11:56:11.926 public.rs:52] Skipping download, using cached copy of file: model.onnx at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-detect-faces-onnx/model.onnx\"\n",
      "INFO 2025-07-30 11:56:11.927 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.932 lib.rs:204] File: tokenizer_config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.932 public.rs:52] Skipping download, using cached copy of file: tokenizer_config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/tokenizer_config.json\"\n",
      "INFO 2025-07-30 11:56:11.933 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.939 lib.rs:204] File: tokenizer.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.939 public.rs:52] Skipping download, using cached copy of file: tokenizer.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.940 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.945 lib.rs:204] File: config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/config.json\"\n",
      "INFO 2025-07-30 11:56:11.945 public.rs:52] Skipping download, using cached copy of file: config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/config.json\"\n",
      "INFO 2025-07-30 11:56:11.946 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.952 lib.rs:204] File: preprocessor_config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/preprocessor_config.json\"\n",
      "INFO 2025-07-30 11:56:11.952 public.rs:52] Skipping download, using cached copy of file: preprocessor_config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/preprocessor_config.json\"\n",
      "INFO 2025-07-30 11:56:11.953 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.959 lib.rs:204] File: tokenizer.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.959 public.rs:52] Skipping download, using cached copy of file: tokenizer.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-prompt-upsampler-trtllm-int8-wo/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:11.960 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.966 lib.rs:204] File: rank0.safetensors found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/rank0.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.966 public.rs:52] Skipping download, using cached copy of file: rank0.safetensors at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/rank0.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.966 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.972 lib.rs:204] File: config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/config.json\"\n",
      "INFO 2025-07-30 11:56:11.972 public.rs:52] Skipping download, using cached copy of file: config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/config.json\"\n",
      "INFO 2025-07-30 11:56:11.973 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.979 lib.rs:204] File: special_tokens_map.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.979 public.rs:52] Skipping download, using cached copy of file: special_tokens_map.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.980 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.986 lib.rs:204] File: lora/adapter_model.safetensors found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/lora/adapter_model.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.986 public.rs:52] Skipping download, using cached copy of file: lora/adapter_model.safetensors at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/lora/adapter_model.safetensors\"\n",
      "INFO 2025-07-30 11:56:11.986 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.992 lib.rs:204] File: special_tokens_map.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.992 public.rs:52] Skipping download, using cached copy of file: special_tokens_map.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-processor/special_tokens_map.json\"\n",
      "INFO 2025-07-30 11:56:11.993 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:11.999 lib.rs:204] File: model_config.yaml found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/model_config.yaml\"\n",
      "INFO 2025-07-30 11:56:11.999 public.rs:52] Skipping download, using cached copy of file: model_config.yaml at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-tokenizer/model_config.yaml\"\n",
      "INFO 2025-07-30 11:56:12.000 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:12.006 lib.rs:204] File: lora/adapter_config.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/lora/adapter_config.json\"\n",
      "INFO 2025-07-30 11:56:12.006 public.rs:52] Skipping download, using cached copy of file: lora/adapter_config.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-aegis-filter-trtllm-int8-sq/lora/adapter_config.json\"\n",
      "INFO 2025-07-30 11:56:12.007 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:12.012 lib.rs:204] File: model.onnx found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-onnx/model.onnx\"\n",
      "INFO 2025-07-30 11:56:12.012 public.rs:52] Skipping download, using cached copy of file: model.onnx at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-siglip-filter-onnx/model.onnx\"\n",
      "INFO 2025-07-30 11:56:12.013 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:12.019 lib.rs:204] File: extra_data.bin found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-onnx-bf16-2/extra_data.bin\"\n",
      "INFO 2025-07-30 11:56:12.019 public.rs:52] Skipping download, using cached copy of file: extra_data.bin at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-onnx-bf16-2/extra_data.bin\"\n",
      "INFO 2025-07-30 11:56:12.020 builder.rs:66] ngc configured with api_loc: api.ngc.nvidia.com auth_loc: authn.nvidia.com scheme: https\n",
      "INFO 2025-07-30 11:56:12.026 lib.rs:204] File: tokenizer.json found in cache: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-tokenizer/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:12.026 public.rs:52] Skipping download, using cached copy of file: tokenizer.json at path: \"/opt/nim/.cache/ngc/hub/models--nim--nvidia--cosmos-predict1-7b-text2world/snapshots/1.0.0-t5-tokenizer/tokenizer.json\"\n",
      "INFO 2025-07-30 11:56:12.026] Using the workspace specified during init: /opt/nim/workspace\n",
      "INFO 2025-07-30 11:56:12.026] Creating workspace at /opt/nim/workspace\n",
      "INFO 2025-07-30 11:56:12.027] Materializing workspace to: /opt/nim/workspace\n",
      "INFO 2025-07-30 11:56:19.511] Results of triton model repository search: {'model_repo': {'t5': {'has_config': True, 'has_version_directory': True}, 't5_tokenizer': {'has_version_directory': True, 'has_config': True}, 'face_blur': {'has_version_directory': True, 'has_config': True}, 'ensemble': {'has_version_directory': True, 'has_config': True}, 'diffusion': {'has_version_directory': True, 'has_config': True}, 'detect_faces': {'has_config': True, 'has_version_directory': True}, 'aegis_filter': {'has_config': True, 'has_version_directory': True}, 'siglip_filter_processor': {'has_version_directory': True, 'has_config': True}, 'siglip_filter': {'has_version_directory': True, 'has_config': True}, 'blocklist_filter': {'has_config': True, 'has_version_directory': True}, 'prompt_upsampler': {'has_version_directory': True, 'has_config': True}}}\n",
      "INFO 2025-07-30 11:56:19.512] Starting Triton server: /opt/nim/.venv/lib/python3.12/site-packages/nimlib/triton_start.sh -m /opt/nim/workspace/model_repo -d - -h 8083 -p 8002 -l 0 -x -\n",
      "INFO 2025-07-30 11:56:19.513] Starting NIM inference server\n",
      "INFO 2025-07-30 11:56:22.559] Running: tritonserver --model-repository /opt/nim/workspace/model_repo --http-port 8083 --metrics-port 8002  --log-verbose 0\n",
      "INFO 2025-07-30 11:56:33.733] INFO 2025-07-30 11:56:33.733] Initializing Aegis\n",
      "INFO 2025-07-30 11:56:39.210] INFO 2025-07-30 11:56:39.210] Initializing Blocklist\n",
      "INFO 2025-07-30 11:56:39.279] INFO 2025-07-30 11:56:39.278] Initializing Blocklist\n",
      "INFO 2025-07-30 11:56:39.307] INFO 2025-07-30 11:56:39.307] Blocklist initializing completed\n",
      "INFO 2025-07-30 11:56:39.334] INFO 2025-07-30 11:56:39.334] Blocklist initializing completed\n",
      "INFO 2025-07-30 11:57:08.617] [TensorRT-LLM] TensorRT-LLM version: 0.18.0.dev2025022500\n",
      "INFO 2025-07-30 11:57:33.752] INFO 2025-07-30 11:57:33.752] Initializing Upsampler\n",
      "INFO 2025-07-30 11:58:00.271] [TensorRT-LLM] TensorRT-LLM version: 0.18.0.dev2025022500\n",
      "ERROR 2025-07-30 12:01:07.357] I0730 11:56:22.790897 278 pinned_memory_manager.cc:277] \"Pinned memory pool is created at '0x7fe65c000000' with size 268435456\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.793367 278 cuda_memory_manager.cc:107] \"CUDA memory pool is created on device 0 with size 67108864\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799375 278 model_lifecycle.cc:473] \"loading: aegis_filter:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799418 278 model_lifecycle.cc:473] \"loading: blocklist_filter:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799452 278 model_lifecycle.cc:473] \"loading: detect_faces:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799485 278 model_lifecycle.cc:473] \"loading: diffusion:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799514 278 model_lifecycle.cc:473] \"loading: ensemble:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799534 278 model_lifecycle.cc:473] \"loading: face_blur:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799552 278 model_lifecycle.cc:473] \"loading: prompt_upsampler:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799576 278 model_lifecycle.cc:473] \"loading: siglip_filter:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799594 278 model_lifecycle.cc:473] \"loading: siglip_filter_processor:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799615 278 model_lifecycle.cc:473] \"loading: t5:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.799635 278 model_lifecycle.cc:473] \"loading: t5_tokenizer:1\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.801363 278 onnxruntime.cc:2880] \"TRITONBACKEND_Initialize: onnxruntime\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.801381 278 onnxruntime.cc:2890] \"Triton TRITONBACKEND API version: 1.19\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.801387 278 onnxruntime.cc:2896] \"'onnxruntime' TRITONBACKEND API version: 1.19\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.801392 278 onnxruntime.cc:2926] \"backend configuration:\\n{\\\"cmdline\\\":{\\\"auto-complete-config\\\":\\\"true\\\",\\\"backend-directory\\\":\\\"/opt/tritonserver/backends\\\",\\\"min-compute-capability\\\":\\\"6.000000\\\",\\\"default-max-batch-size\\\":\\\"4\\\"}}\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.824022 278 onnxruntime.cc:2991] \"TRITONBACKEND_ModelInitialize: detect_faces (version 1)\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.824490 278 onnxruntime.cc:989] \"skipping model configuration auto-complete for 'detect_faces': inputs and outputs already specified\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.824554 278 python_be.cc:1845] \"Input tensors can be both in CPU and GPU. FORCE_CPU_ONLY_INPUT_TENSORS is off.\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:22.825231 278 onnxruntime.cc:3056] \"TRITONBACKEND_ModelInstanceInitialize: detect_faces_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/cross_entropy.py:160: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.358] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/parallel_state.py:123: SyntaxWarning: invalid escape sequence '\\i'\n",
      "ERROR 2025-07-30 12:01:07.358] \"\"\"Generate orthogonal parallel groups based on the parallel size and mask.\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:28.883408 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: aegis_filter_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:31.709086 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: blocklist_filter_0_1 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:31.709153 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: blocklist_filter_0_0 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:39.336235 278 model_lifecycle.cc:849] \"successfully loaded 'blocklist_filter'\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1776: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.358] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1950: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.358] command_line, option=\"strong\", grp_pattern=\"\\d+\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/cross_entropy.py:160: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.358] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/parallel_state.py:123: SyntaxWarning: invalid escape sequence '\\i'\n",
      "ERROR 2025-07-30 12:01:07.358] \"\"\"Generate orthogonal parallel groups based on the parallel size and mask.\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:289: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:300: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:392: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:432: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:44.245681 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: ensemble_0_1 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:44.245772 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: ensemble_0_0 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.358] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.358] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.358] I0730 11:56:47.628813 278 model_lifecycle.cc:849] \"successfully loaded 'ensemble'\"\n",
      "ERROR 2025-07-30 12:01:07.359] [NeMo W 2025-07-30 11:56:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/tensor_parallel/cross_entropy.py:78: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.359] \n",
      "ERROR 2025-07-30 12:01:07.359] [NeMo W 2025-07-30 11:56:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_with_interleaving.py:49: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"Run interleaved 1F1B schedule with communication between pipeline stages as needed.\n",
      "ERROR 2025-07-30 12:01:07.359] \n",
      "ERROR 2025-07-30 12:01:07.359] [NeMo W 2025-07-30 11:56:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_without_interleaving.py:261: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"Run non-interleaved 1F1B schedule, with communication between pipeline stages.\n",
      "ERROR 2025-07-30 12:01:07.359] \n",
      "ERROR 2025-07-30 12:01:07.359] [NeMo W 2025-07-30 11:56:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.359] \n",
      "ERROR 2025-07-30 12:01:07.359] [NeMo W 2025-07-30 11:56:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/einops/einops.py:827: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.359] \n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:289: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.359] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:300: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.359] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:392: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.359] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:432: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.359] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/tensor_parallel/cross_entropy.py:78: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_with_interleaving.py:49: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.359] \"\"\"Run interleaved 1F1B schedule with communication between pipeline stages as needed.\n",
      "ERROR 2025-07-30 12:01:07.359] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_without_interleaving.py:261: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.360] \"\"\"Run non-interleaved 1F1B schedule, with communication between pipeline stages.\n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/llm/peft/utils.py:75: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.360] return re.match('.*mlp\\.experts\\.local_experts.[0-9]+\\.linear_fc[1-2]$', fqn) is not None\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] /opt/nim/.venv/lib/python3.12/site-packages/einops/einops.py:827: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:107: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?½\", \"\\\\1 1/2\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:108: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?⅓\", \"\\\\1 1/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:109: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?⅔\", \"\\\\1 2/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:110: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?¼\", \"\\\\1 1/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:111: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?¾\", \"\\\\1 3/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:112: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?⅕\", \"\\\\1 1/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:113: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?⅖\", \"\\\\1 2/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.360] \n",
      "ERROR 2025-07-30 12:01:07.360] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:114: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.360] written = re.sub(\"([0-9])\\s?⅗\", \"\\\\1 3/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:115: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅘\", \"\\\\1 4/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:116: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅙\", \"\\\\1 1/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:117: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅚\", \"\\\\1 5/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:118: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅛\", \"\\\\1 1/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:119: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅜\", \"\\\\1 3/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:120: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅝\", \"\\\\1 5/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:121: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.361] written = re.sub(\"([0-9])\\s?⅞\", \"\\\\1 7/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/bert_module.py:57: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.361] m = re.match(\"^bert.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/gpt_module.py:68: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.361] m = re.match(\"^gpt.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] I0730 11:56:56.962814 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: face_blur_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.361] I0730 11:56:56.962852 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: face_blur_0_1 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.361] I0730 11:56:56.962914 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: face_blur_0_3 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.361] I0730 11:56:56.963021 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: face_blur_0_2 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:57 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.361] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:57 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1776: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.361] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:57 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1950: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.361] command_line, option=\"strong\", grp_pattern=\"\\d+\"\n",
      "ERROR 2025-07-30 12:01:07.361] \n",
      "ERROR 2025-07-30 12:01:07.361] [NeMo W 2025-07-30 11:56:58 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.362] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:58 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.362] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.362] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.362] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.362] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:44: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.362] re_han_default = re.compile(\"([\\u4E00-\\u9FD5a-zA-Z0-9+#&\\._%\\-]+)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:46: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] re_skip_default = re.compile(\"(\\r\\n|\\s)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:56:59 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/finalseg/__init__.py:78: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.362] re_skip = re.compile(\"([a-zA-Z0-9]+(?:\\.\\d+)?%?)\")\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:00 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/megatron/vocab_parallel_cross_entropy.py:88: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:00 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.362] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:00 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/llm/peft/utils.py:75: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.362] return re.match('.*mlp\\.experts\\.local_experts.[0-9]+\\.linear_fc[1-2]$', fqn) is not None\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] The package `megatron.core` was not imported in this environment which is needed for SSMs.\n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:107: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?½\", \"\\\\1 1/2\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:108: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?⅓\", \"\\\\1 1/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:109: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?⅔\", \"\\\\1 2/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:110: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?¼\", \"\\\\1 1/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:111: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?¾\", \"\\\\1 3/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:112: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?⅕\", \"\\\\1 1/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:113: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.362] written = re.sub(\"([0-9])\\s?⅖\", \"\\\\1 2/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.362] \n",
      "ERROR 2025-07-30 12:01:07.362] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:114: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅗\", \"\\\\1 3/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:115: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅘\", \"\\\\1 4/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:116: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅙\", \"\\\\1 1/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:117: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅚\", \"\\\\1 5/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:118: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅛\", \"\\\\1 1/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:119: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅜\", \"\\\\1 3/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:120: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅝\", \"\\\\1 5/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:121: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] written = re.sub(\"([0-9])\\s?⅞\", \"\\\\1 7/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/bert_module.py:57: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.363] m = re.match(\"^bert.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/mcore-vfm/megatron/training/arguments.py:1090: SyntaxWarning: invalid escape sequence '\\ '\n",
      "ERROR 2025-07-30 12:01:07.363] '   --rampup-batch-size 16 8 300000 \\ '\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:02 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/gpt_module.py:68: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.363] m = re.match(\"^gpt.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:03 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.363] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.363] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.363] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.363] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:44: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.363] re_han_default = re.compile(\"([\\u4E00-\\u9FD5a-zA-Z0-9+#&\\._%\\-]+)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:46: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] re_skip_default = re.compile(\"(\\r\\n|\\s)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:04 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/finalseg/__init__.py:78: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.363] re_skip = re.compile(\"([a-zA-Z0-9]+(?:\\.\\d+)?%?)\")\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] [NeMo W 2025-07-30 11:57:05 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/megatron/vocab_parallel_cross_entropy.py:88: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.363] \n",
      "ERROR 2025-07-30 12:01:07.363] I0730 11:57:06.016319 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: diffusion_0_0 (MODEL device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.363] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/cross_entropy.py:160: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.363] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:06 nemo_logging:405] The package `megatron.core` was not imported in this environment which is needed for SSMs.\n",
      "ERROR 2025-07-30 12:01:07.364] I0730 11:57:07.018080 278 model_lifecycle.cc:849] \"successfully loaded 'face_blur'\"\n",
      "ERROR 2025-07-30 12:01:07.364] 2025-07-30 11:57:08,533 - INFO - flashinfer.jit: Prebuilt kernels not found, using JIT backend\n",
      "ERROR 2025-07-30 12:01:07.364] Loading Model: \u001b[1;32m[1/2]\t\u001b[0mLoading TRT checkpoints to memory\n",
      "ERROR 2025-07-30 12:01:07.364] \u001b[38;20mTime: 0.447s\n",
      "ERROR 2025-07-30 12:01:07.364] \u001b[0mLoading Model: \u001b[1;32m[2/2]\t\u001b[0mBuild TRT-LLM engine\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/parallel_state.py:123: SyntaxWarning: invalid escape sequence '\\i'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"Generate orthogonal parallel groups based on the parallel size and mask.\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.364] I0730 11:57:21.484044 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: prompt_upsampler_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:289: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:300: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:392: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:432: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.364] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:34 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/tensor_parallel/cross_entropy.py:78: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:34 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_with_interleaving.py:49: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"Run interleaved 1F1B schedule with communication between pipeline stages as needed.\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:34 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_without_interleaving.py:261: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"Run non-interleaved 1F1B schedule, with communication between pipeline stages.\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:34 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:34 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/einops/einops.py:827: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1776: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1950: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.364] command_line, option=\"strong\", grp_pattern=\"\\d+\"\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/cross_entropy.py:160: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.364] /opt/nim/third_party/mcore-vfm/megatron/core/parallel_state.py:123: SyntaxWarning: invalid escape sequence '\\i'\n",
      "ERROR 2025-07-30 12:01:07.364] \"\"\"Generate orthogonal parallel groups based on the parallel size and mask.\n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:37 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/llm/peft/utils.py:75: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.364] return re.match('.*mlp\\.experts\\.local_experts.[0-9]+\\.linear_fc[1-2]$', fqn) is not None\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:107: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.364] written = re.sub(\"([0-9])\\s?½\", \"\\\\1 1/2\", written)\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.364] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:108: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.364] written = re.sub(\"([0-9])\\s?⅓\", \"\\\\1 1/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.364] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:109: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅔\", \"\\\\1 2/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:110: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?¼\", \"\\\\1 1/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:111: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?¾\", \"\\\\1 3/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:112: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅕\", \"\\\\1 1/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:113: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅖\", \"\\\\1 2/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:114: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅗\", \"\\\\1 3/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:115: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅘\", \"\\\\1 4/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:116: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅙\", \"\\\\1 1/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:117: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅚\", \"\\\\1 5/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:118: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅛\", \"\\\\1 1/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:119: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅜\", \"\\\\1 3/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:120: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅝\", \"\\\\1 5/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:121: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.365] written = re.sub(\"([0-9])\\s?⅞\", \"\\\\1 7/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:38 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/bert_module.py:57: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.365] m = re.match(\"^bert.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:39 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/gpt_module.py:68: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.365] m = re.match(\"^gpt.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:39 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.365] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:40 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1776: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.365] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:40 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/pulp/tests/test_pulp.py:1950: SyntaxWarning: invalid escape sequence '\\d'\n",
      "ERROR 2025-07-30 12:01:07.365] command_line, option=\"strong\", grp_pattern=\"\\d+\"\n",
      "ERROR 2025-07-30 12:01:07.365] \n",
      "ERROR 2025-07-30 12:01:07.365] [NeMo W 2025-07-30 11:57:41 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.365] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:41 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.366] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.366] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.366] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:44: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.366] re_han_default = re.compile(\"([\\u4E00-\\u9FD5a-zA-Z0-9+#&\\._%\\-]+)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:46: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] re_skip_default = re.compile(\"(\\r\\n|\\s)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/finalseg/__init__.py:78: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.366] re_skip = re.compile(\"([a-zA-Z0-9]+(?:\\.\\d+)?%?)\")\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:42 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/megatron/vocab_parallel_cross_entropy.py:88: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:45 nemo_logging:405] The package `megatron.core` was not imported in this environment which is needed for SSMs.\n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:45 nemo_logging:405] /opt/nim/third_party/mcore-vfm/megatron/training/arguments.py:1090: SyntaxWarning: invalid escape sequence '\\ '\n",
      "ERROR 2025-07-30 12:01:07.366] '   --rampup-batch-size 16 8 300000 \\ '\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:289: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.366] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:300: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.366] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:392: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.366] @custom_fwd\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/third_party/mcore-vfm/megatron/core/tensor_parallel/layers.py:432: FutureWarning: `torch.cuda.amp.custom_bwd(args...)` is deprecated. Please use `torch.amp.custom_bwd(args..., device_type='cuda')` instead.\n",
      "ERROR 2025-07-30 12:01:07.366] @custom_bwd\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/tensor_parallel/cross_entropy.py:78: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_with_interleaving.py:49: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"Run interleaved 1F1B schedule with communication between pipeline stages as needed.\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/.venv/lib/python3.12/site-packages/apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_without_interleaving.py:261: SyntaxWarning: invalid escape sequence '\\_'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"Run non-interleaved 1F1B schedule, with communication between pipeline stages.\n",
      "ERROR 2025-07-30 12:01:07.366] /opt/nim/.venv/lib/python3.12/site-packages/einops/einops.py:827: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:51 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.366] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:52 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/llm/peft/utils.py:75: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.366] return re.match('.*mlp\\.experts\\.local_experts.[0-9]+\\.linear_fc[1-2]$', fqn) is not None\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:107: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] written = re.sub(\"([0-9])\\s?½\", \"\\\\1 1/2\", written)\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:108: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.366] written = re.sub(\"([0-9])\\s?⅓\", \"\\\\1 1/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.366] \n",
      "ERROR 2025-07-30 12:01:07.366] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:109: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅔\", \"\\\\1 2/3\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:110: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?¼\", \"\\\\1 1/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:111: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?¾\", \"\\\\1 3/4\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:112: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅕\", \"\\\\1 1/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:113: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅖\", \"\\\\1 2/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:114: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅗\", \"\\\\1 3/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:115: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅘\", \"\\\\1 4/5\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:116: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅙\", \"\\\\1 1/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:117: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅚\", \"\\\\1 5/6\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:118: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅛\", \"\\\\1 1/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:119: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅜\", \"\\\\1 3/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:120: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅝\", \"\\\\1 5/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/data/text_normalization/utils.py:121: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.367] written = re.sub(\"([0-9])\\s?⅞\", \"\\\\1 7/8\", written)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/bert_module.py:57: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.367] m = re.match(\"^bert.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/gpt_module.py:68: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.367] m = re.match(\"^gpt.*?\\.\", list(pretrained_dict.keys())[0])\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:54 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.367] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:55 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py:181: SyntaxWarning: invalid escape sequence '\\{'\n",
      "ERROR 2025-07-30 12:01:07.367] \"prompt_template_fields\": re.findall(\"\\{(.*?)\\}\", task.prompt_template),\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:55 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/models/language_modeling/megatron_base_model.py:390: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.367] return re.fullmatch(\"[0-9][0-9]\\.[0-9][0-9].*\", nvidia_torch_version)  # \"YY.MM.*\"\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:55 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/apex/contrib/optimizers/distributed_fused_adam.py:274: SyntaxWarning: invalid escape sequence '\\:'\n",
      "ERROR 2025-07-30 12:01:07.367] \"\"\"Adam optimizer with ZeRO algorithm.\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.367] [NeMo W 2025-07-30 11:57:56 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:44: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.367] re_han_default = re.compile(\"([\\u4E00-\\u9FD5a-zA-Z0-9+#&\\._%\\-]+)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.367] \n",
      "ERROR 2025-07-30 12:01:07.368] [NeMo W 2025-07-30 11:57:56 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/__init__.py:46: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.368] re_skip_default = re.compile(\"(\\r\\n|\\s)\", re.U)\n",
      "ERROR 2025-07-30 12:01:07.368] \n",
      "ERROR 2025-07-30 12:01:07.368] [NeMo W 2025-07-30 11:57:56 nemo_logging:405] /opt/nim/.venv/lib/python3.12/site-packages/jieba/finalseg/__init__.py:78: SyntaxWarning: invalid escape sequence '\\.'\n",
      "ERROR 2025-07-30 12:01:07.368] re_skip = re.compile(\"([a-zA-Z0-9]+(?:\\.\\d+)?%?)\")\n",
      "ERROR 2025-07-30 12:01:07.368] \n",
      "ERROR 2025-07-30 12:01:07.368] [NeMo W 2025-07-30 11:57:56 nemo_logging:405] /opt/nim/third_party/nemo-vfm/nemo/collections/nlp/modules/common/megatron/vocab_parallel_cross_entropy.py:88: SyntaxWarning: invalid escape sequence '\\s'\n",
      "ERROR 2025-07-30 12:01:07.368] \"\"\"\n",
      "ERROR 2025-07-30 12:01:07.368] \n",
      "ERROR 2025-07-30 12:01:07.368] [NeMo W 2025-07-30 11:57:58 nemo_logging:405] The package `megatron.core` was not imported in this environment which is needed for SSMs.\n",
      "ERROR 2025-07-30 12:01:07.368] 2025-07-30 11:57:59,873 - INFO - flashinfer.jit: Prebuilt kernels not found, using JIT backend\n",
      "ERROR 2025-07-30 12:01:07.368] Loading Model: \u001b[1;32m[1/2]\t\u001b[0mLoading TRT checkpoints to memory\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[38;20mTime: 0.557s\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0mLoading Model: \u001b[1;32m[2/2]\t\u001b[0mBuild TRT-LLM engine\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[38;20mTime: 58.669s\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0m\u001b[1;32mLoading model done.\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0m\u001b[38;20mTotal latency: 59.116s\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0m\u001b[33;20mUsing MpiPoolSession to spawn MPI processes\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0mI0730 11:58:28.421817 278 pb_stub.cc:366] \"Failed to initialize Python stub: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /usr/lib/python3.12/tarfile.py(254): copyfileobj\\n  /usr/lib/python3.12/tarfile.py(2502): makefile\\n  /usr/lib/python3.12/tarfile.py(2448): _extract_member\\n  /usr/lib/python3.12/tarfile.py(2369): _extract_one\\n  /usr/lib/python3.12/tarfile.py(2302): extractall\\n  /opt/nim/workspace/model_repo/diffusion/1/model.py(212): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:28.784994 278 pb_stub.cc:366] \"Failed to initialize Python stub: OSError: [Errno 28] No space left on device: '/tmp/tmpopbmwfm1'\\n\\nAt:\\n  /usr/lib/python3.12/concurrent/futures/_base.py(404): __get_result\\n  /usr/lib/python3.12/concurrent/futures/_base.py(461): result\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(368): shutdown\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(220): _handle_background_error\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(307): _start_executor_workers\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(91): __init__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(336): create\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(501): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/aegis_filter/1/model.py(102): initialize\\n  /usr/lib/python3.12/contextlib.py(81): inner\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] E0730 11:58:30.451753 278 backend_model.cc:692] \"ERROR: Failed to create instance: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /usr/lib/python3.12/tarfile.py(254): copyfileobj\\n  /usr/lib/python3.12/tarfile.py(2502): makefile\\n  /usr/lib/python3.12/tarfile.py(2448): _extract_member\\n  /usr/lib/python3.12/tarfile.py(2369): _extract_one\\n  /usr/lib/python3.12/tarfile.py(2302): extractall\\n  /opt/nim/workspace/model_repo/diffusion/1/model.py(212): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] E0730 11:58:30.454244 278 model_lifecycle.cc:654] \"failed to load 'diffusion' version 1: Internal: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /usr/lib/python3.12/tarfile.py(254): copyfileobj\\n  /usr/lib/python3.12/tarfile.py(2502): makefile\\n  /usr/lib/python3.12/tarfile.py(2448): _extract_member\\n  /usr/lib/python3.12/tarfile.py(2369): _extract_one\\n  /usr/lib/python3.12/tarfile.py(2302): extractall\\n  /opt/nim/workspace/model_repo/diffusion/1/model.py(212): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:30.454324 278 model_lifecycle.cc:789] \"failed to load 'diffusion'\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:30.537977 278 onnxruntime.cc:2991] \"TRITONBACKEND_ModelInitialize: siglip_filter (version 1)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:30.538463 278 onnxruntime.cc:989] \"skipping model configuration auto-complete for 'siglip_filter': inputs and outputs already specified\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:30.539234 278 onnxruntime.cc:3056] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[38;20mTime: 43.457s\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0m\u001b[1;32mLoading model done.\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0m\u001b[38;20mTotal latency: 44.013s\n",
      "ERROR 2025-07-30 12:01:07.368] \u001b[0mI0730 11:58:45.222751 278 pb_stub.cc:366] \"Failed to initialize Python stub: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/_common.py(119): serialize_engine\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/builder.py(763): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1386): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1337): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1804): build_task\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1850): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1721): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(418): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/prompt_upsampler/1/model.py(99): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] E0730 11:58:47.385311 278 backend_model.cc:692] \"ERROR: Failed to create instance: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/_common.py(119): serialize_engine\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/builder.py(763): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1386): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1337): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1804): build_task\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1850): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1721): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(418): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/prompt_upsampler/1/model.py(99): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] E0730 11:58:47.385403 278 model_lifecycle.cc:654] \"failed to load 'prompt_upsampler' version 1: Internal: OSError: [Errno 28] No space left on device\\n\\nAt:\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/_common.py(119): serialize_engine\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/builder.py(763): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1386): save\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1337): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1804): build_task\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1850): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1721): __call__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(418): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/prompt_upsampler/1/model.py(99): initialize\\n\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:47.385427 278 model_lifecycle.cc:789] \"failed to load 'prompt_upsampler'\"\n",
      "ERROR 2025-07-30 12:01:07.368] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.368] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337306 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_0 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337398 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_3 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337365 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_2 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337338 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_1 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337428 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_4 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337494 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_5 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.337530 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_6 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.338429 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_7 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.338606 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_10 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.338653 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_8 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.338662 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_9 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.339000 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_11 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.339599 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_12 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.340865 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_13 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.341248 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_14 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] I0730 11:58:58.341758 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: siglip_filter_processor_0_15 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.368] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.368] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.368] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.368] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.368] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.368] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.368] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:24.541552 278 model_lifecycle.cc:849] \"successfully loaded 'siglip_filter_processor'\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:24.544390 278 onnxruntime.cc:2991] \"TRITONBACKEND_ModelInitialize: t5 (version 1)\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:24.544808 278 onnxruntime.cc:989] \"skipping model configuration auto-complete for 't5': inputs and outputs already specified\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:24.545397 278 onnxruntime.cc:3056] \"TRITONBACKEND_ModelInstanceInitialize: t5_0_0 (GPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:38.313351 278 model_lifecycle.cc:849] \"successfully loaded 'detect_faces'\"\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:52.348804 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: t5_tokenizer_0_0 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 11:59:52.348854 278 python_be.cc:2249] \"TRITONBACKEND_ModelInstanceInitialize: t5_tokenizer_0_1 (CPU device 0)\"\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] /opt/nim/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.\n",
      "ERROR 2025-07-30 12:01:07.369] warnings.warn(\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:04.989898 278 model_lifecycle.cc:849] \"successfully loaded 't5_tokenizer'\"\n",
      "ERROR 2025-07-30 12:01:07.369] E0730 12:00:23.839999 278 backend_model.cc:692] \"ERROR: Failed to create instance: OSError: [Errno 28] No space left on device: '/tmp/tmpopbmwfm1'\\n\\nAt:\\n  /usr/lib/python3.12/concurrent/futures/_base.py(404): __get_result\\n  /usr/lib/python3.12/concurrent/futures/_base.py(461): result\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(368): shutdown\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(220): _handle_background_error\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(307): _start_executor_workers\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(91): __init__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(336): create\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(501): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/aegis_filter/1/model.py(102): initialize\\n  /usr/lib/python3.12/contextlib.py(81): inner\\n\"\n",
      "ERROR 2025-07-30 12:01:07.369] E0730 12:00:23.840091 278 model_lifecycle.cc:654] \"failed to load 'aegis_filter' version 1: Internal: OSError: [Errno 28] No space left on device: '/tmp/tmpopbmwfm1'\\n\\nAt:\\n  /usr/lib/python3.12/concurrent/futures/_base.py(404): __get_result\\n  /usr/lib/python3.12/concurrent/futures/_base.py(461): result\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(368): shutdown\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(220): _handle_background_error\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(307): _start_executor_workers\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(91): __init__\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(336): create\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(501): _build_model\\n  /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__\\n  /opt/nim/workspace/model_repo/aegis_filter/1/model.py(102): initialize\\n  /usr/lib/python3.12/contextlib.py(81): inner\\n\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:23.840116 278 model_lifecycle.cc:789] \"failed to load 'aegis_filter'\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:34.611531 278 model_lifecycle.cc:849] \"successfully loaded 'siglip_filter'\"\n",
      "ERROR 2025-07-30 12:01:07.369] \u001b[0;93m2025-07-30 12:00:34.650572581 [W:onnxruntime:, transformer_memcpy.cc:74 ApplyImpl] 1 Memcpy nodes are added to the graph main_graph for CUDAExecutionProvider. It might have negative impact on performance (including unable to run CUDA graph). Set session_options.log_severity_level=1 to see the detail logs before this message.\u001b[m\n",
      "ERROR 2025-07-30 12:01:07.369] \u001b[0;93m2025-07-30 12:00:34.655390171 [W:onnxruntime:, session_state.cc:1168 VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.\u001b[m\n",
      "ERROR 2025-07-30 12:01:07.369] \u001b[0;93m2025-07-30 12:00:34.655404974 [W:onnxruntime:, session_state.cc:1170 VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.\u001b[m\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:36.836103 278 model_lifecycle.cc:849] \"successfully loaded 't5'\"\n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:36.836304 278 server.cc:604]\n",
      "ERROR 2025-07-30 12:01:07.369] +------------------+------+\n",
      "ERROR 2025-07-30 12:01:07.369] | Repository Agent | Path |\n",
      "ERROR 2025-07-30 12:01:07.369] +------------------+------+\n",
      "ERROR 2025-07-30 12:01:07.369] +------------------+------+\n",
      "ERROR 2025-07-30 12:01:07.369] \n",
      "ERROR 2025-07-30 12:01:07.369] I0730 12:00:36.836348 278 server.cc:631]\n",
      "ERROR 2025-07-30 12:01:07.369] +-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.369] | Backend     | Path                                                            | Config                                                                                                                                                        |\n",
      "ERROR 2025-07-30 12:01:07.369] +-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.369] | python      | /opt/tritonserver/backends/python/libtriton_python.so           | {\"cmdline\":{\"auto-complete-config\":\"true\",\"backend-directory\":\"/opt/tritonserver/backends\",\"min-compute-capability\":\"6.000000\",\"default-max-batch-size\":\"4\"}} |\n",
      "ERROR 2025-07-30 12:01:07.369] | onnxruntime | /opt/tritonserver/backends/onnxruntime/libtriton_onnxruntime.so | {\"cmdline\":{\"auto-complete-config\":\"true\",\"backend-directory\":\"/opt/tritonserver/backends\",\"min-compute-capability\":\"6.000000\",\"default-max-batch-size\":\"4\"}} |\n",
      "ERROR 2025-07-30 12:01:07.370] +-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] \n",
      "ERROR 2025-07-30 12:01:07.370] I0730 12:00:36.836445 278 server.cc:674]\n",
      "ERROR 2025-07-30 12:01:07.370] +-------------------------+---------+----------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] | Model                   | Version | Status                                                                                                         |\n",
      "ERROR 2025-07-30 12:01:07.370] +-------------------------+---------+----------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] | aegis_filter            | 1       | UNAVAILABLE: Internal: OSError: [Errno 28] No space left on device: '/tmp/tmpopbmwfm1'                         |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |                                                                                                                |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         | At:                                                                                                            |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/concurrent/futures/_base.py(404): __get_result                                           |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/concurrent/futures/_base.py(461): result                                                 |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(368): shutdown                    |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(220): _handle_background_error |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(307): _start_executor_workers     |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/proxy.py(91): __init__                     |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/executor/executor.py(336): create                   |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(501): _build_model                    |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__                        |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/workspace/model_repo/aegis_filter/1/model.py(102): initialize                                       |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/contextlib.py(81): inner                                                                 |\n",
      "ERROR 2025-07-30 12:01:07.370] | blocklist_filter        | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | detect_faces            | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | diffusion               | 1       | UNAVAILABLE: Internal: OSError: [Errno 28] No space left on device                                             |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |                                                                                                                |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         | At:                                                                                                            |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/tarfile.py(254): copyfileobj                                                             |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/tarfile.py(2502): makefile                                                               |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/tarfile.py(2448): _extract_member                                                        |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/tarfile.py(2369): _extract_one                                                           |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /usr/lib/python3.12/tarfile.py(2302): extractall                                                             |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/workspace/model_repo/diffusion/1/model.py(212): initialize                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | ensemble                | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | face_blur               | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | prompt_upsampler        | 1       | UNAVAILABLE: Internal: OSError: [Errno 28] No space left on device                                             |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |                                                                                                                |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         | At:                                                                                                            |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/_common.py(119): serialize_engine                   |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/builder.py(763): save                               |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1386): save                     |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1337): __call__                 |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1804): build_task               |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1850): _build_model             |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm_utils.py(1721): __call__                 |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(418): _build_model                    |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/.venv/lib/python3.12/site-packages/tensorrt_llm/llmapi/llm.py(165): __init__                        |\n",
      "ERROR 2025-07-30 12:01:07.370] |                         |         |   /opt/nim/workspace/model_repo/prompt_upsampler/1/model.py(99): initialize                                    |\n",
      "ERROR 2025-07-30 12:01:07.370] | siglip_filter           | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | siglip_filter_processor | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | t5                      | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] | t5_tokenizer            | 1       | READY                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.370] +-------------------------+---------+----------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] \n",
      "ERROR 2025-07-30 12:01:07.370] I0730 12:00:36.919870 278 metrics.cc:890] \"Collecting metrics for GPU 0: NVIDIA A100 80GB PCIe\"\n",
      "ERROR 2025-07-30 12:01:07.370] I0730 12:00:36.929284 278 metrics.cc:783] \"Collecting CPU metrics\"\n",
      "ERROR 2025-07-30 12:01:07.370] I0730 12:00:36.929379 278 tritonserver.cc:2598]\n",
      "ERROR 2025-07-30 12:01:07.370] +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] | Option                           | Value                                                                                                                                                                                                           |\n",
      "ERROR 2025-07-30 12:01:07.370] +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.370] | server_id                        | triton                                                                                                                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.371] | server_version                   | 2.54.0                                                                                                                                                                                                          |\n",
      "ERROR 2025-07-30 12:01:07.371] | server_extensions                | classification sequence model_repository model_repository(unload_dependents) schedule_policy model_configuration system_shared_memory cuda_shared_memory binary_tensor_data parameters statistics trace logging |\n",
      "ERROR 2025-07-30 12:01:07.371] | model_repository_path[0]         | /opt/nim/workspace/model_repo                                                                                                                                                                                   |\n",
      "ERROR 2025-07-30 12:01:07.371] | model_control_mode               | MODE_NONE                                                                                                                                                                                                       |\n",
      "ERROR 2025-07-30 12:01:07.371] | strict_model_config              | 0                                                                                                                                                                                                               |\n",
      "ERROR 2025-07-30 12:01:07.371] | model_config_name                |                                                                                                                                                                                                                 |\n",
      "ERROR 2025-07-30 12:01:07.371] | rate_limit                       | OFF                                                                                                                                                                                                             |\n",
      "ERROR 2025-07-30 12:01:07.371] | pinned_memory_pool_byte_size     | 268435456                                                                                                                                                                                                       |\n",
      "ERROR 2025-07-30 12:01:07.371] | cuda_memory_pool_byte_size{0}    | 67108864                                                                                                                                                                                                        |\n",
      "ERROR 2025-07-30 12:01:07.371] | min_supported_compute_capability | 6.0                                                                                                                                                                                                             |\n",
      "ERROR 2025-07-30 12:01:07.371] | strict_readiness                 | 1                                                                                                                                                                                                               |\n",
      "ERROR 2025-07-30 12:01:07.371] | exit_timeout                     | 30                                                                                                                                                                                                              |\n",
      "ERROR 2025-07-30 12:01:07.371] | cache_enabled                    | 0                                                                                                                                                                                                               |\n",
      "ERROR 2025-07-30 12:01:07.371] +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+\n",
      "ERROR 2025-07-30 12:01:07.371] \n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.929452 278 server.cc:305] \"Waiting for in-flight requests to complete.\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.929464 278 server.cc:321] \"Timeout 30: Found 0 model versions that have in-flight inferences\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.930049 278 onnxruntime.cc:3108] \"TRITONBACKEND_ModelInstanceFinalize: delete instance state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.930080 278 server.cc:336] \"All models are stopped, unloading models\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.930199 278 server.cc:345] \"Timeout 30: Found 8 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.930254 278 onnxruntime.cc:3108] \"TRITONBACKEND_ModelInstanceFinalize: delete instance state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.930328 278 onnxruntime.cc:3108] \"TRITONBACKEND_ModelInstanceFinalize: delete instance state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.976315 278 onnxruntime.cc:3032] \"TRITONBACKEND_ModelFinalize: delete model state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.976380 278 model_lifecycle.cc:636] \"successfully unloaded 't5' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.992725 278 onnxruntime.cc:3032] \"TRITONBACKEND_ModelFinalize: delete model state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:36.992909 278 model_lifecycle.cc:636] \"successfully unloaded 'detect_faces' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:37.301325 278 onnxruntime.cc:3032] \"TRITONBACKEND_ModelFinalize: delete model state\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:37.301418 278 model_lifecycle.cc:636] \"successfully unloaded 'siglip_filter' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:37.930520 278 server.cc:345] \"Timeout 29: Found 5 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:38.930645 278 server.cc:345] \"Timeout 28: Found 5 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:39.903026 278 model_lifecycle.cc:636] \"successfully unloaded 'blocklist_filter' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:39.930753 278 server.cc:345] \"Timeout 27: Found 4 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:39.960279 278 model_lifecycle.cc:636] \"successfully unloaded 'ensemble' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:40.768333 278 model_lifecycle.cc:636] \"successfully unloaded 't5_tokenizer' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:40.930865 278 server.cc:345] \"Timeout 26: Found 2 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:41.930995 278 server.cc:345] \"Timeout 25: Found 2 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:42.931121 278 server.cc:345] \"Timeout 24: Found 2 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:43.931230 278 server.cc:345] \"Timeout 23: Found 2 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:44.008994 278 model_lifecycle.cc:636] \"successfully unloaded 'face_blur' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:44.931357 278 server.cc:345] \"Timeout 22: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:45.931470 278 server.cc:345] \"Timeout 21: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:46.931598 278 server.cc:345] \"Timeout 20: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:47.931724 278 server.cc:345] \"Timeout 19: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:48.931832 278 server.cc:345] \"Timeout 18: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:49.931962 278 server.cc:345] \"Timeout 17: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:50.932091 278 server.cc:345] \"Timeout 16: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:51.932195 278 server.cc:345] \"Timeout 15: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:52.932326 278 server.cc:345] \"Timeout 14: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:53.932436 278 server.cc:345] \"Timeout 13: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:54.932558 278 server.cc:345] \"Timeout 12: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:55.932686 278 server.cc:345] \"Timeout 11: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:56.932813 278 server.cc:345] \"Timeout 10: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:57.932935 278 server.cc:345] \"Timeout 9: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.371] I0730 12:00:58.933062 278 server.cc:345] \"Timeout 8: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:00:59.933182 278 server.cc:345] \"Timeout 7: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:00.933314 278 server.cc:345] \"Timeout 6: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:01.933430 278 server.cc:345] \"Timeout 5: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:02.933624 278 server.cc:345] \"Timeout 4: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:03.933759 278 server.cc:345] \"Timeout 3: Found 1 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:04.840970 278 model_lifecycle.cc:636] \"successfully unloaded 'siglip_filter_processor' version 1\"\n",
      "ERROR 2025-07-30 12:01:07.372] I0730 12:01:04.933880 278 server.cc:345] \"Timeout 2: Found 0 live models and 0 in-flight non-inference requests\"\n",
      "ERROR 2025-07-30 12:01:07.372] error: creating server: Internal - failed to load all models\n",
      "INFO 2025-07-30 12:26:20.371] Serving endpoints:\n",
      "  0.0.0.0:8000/v1/health/live (GET)\n",
      "  0.0.0.0:8000/v1/health/ready (GET)\n",
      "  0.0.0.0:8000/v1/metrics (GET)\n",
      "  0.0.0.0:8000/v1/license (GET)\n",
      "  0.0.0.0:8000/v1/metadata (GET)\n",
      "  0.0.0.0:8000/v1/manifest (GET)\n",
      "  0.0.0.0:8000/v1/infer (POST)\n",
      "INFO 2025-07-30 12:26:20.371] {'message': 'Starting HTTP Inference server', 'port': 8000, 'workers_count': 1, 'host': '0.0.0.0', 'log_level': 'info', 'SSL': 'disabled'}\n"
     ]
    }
   ],
   "source": [
    "!docker logs cosmos-predict1-7b-text2world"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "b1f6a6d3-8572-4c6f-9eea-c4205aabffa6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP/1.1 503 Service Unavailable\n",
      "\u001b[1mdate\u001b[0m: Wed, 30 Jul 2025 12:41:11 GMT\n",
      "\u001b[1mserver\u001b[0m: uvicorn\n",
      "\u001b[1mcontent-length\u001b[0m: 233\n",
      "\u001b[1mcontent-type\u001b[0m: application/json\n",
      "\n",
      "{\"description\":\"Triton liveness check\",\"status\":\"error\",\"error\":\"[StatusCode.UNAVAILABLE] failed to connect to all addresses; last error: UNKNOWN: ipv6:%5B::1%5D:8001: Failed to connect to remote host: Timeout occurred: FD Shutdown\"}"
     ]
    }
   ],
   "source": [
    "!curl -i -X GET 'http://0.0.0.0:8000/v1/health/live'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3a9d448d-104d-4bce-b29e-84337a96fd9a",
   "metadata": {},
   "source": [
    "## Start Inferencing\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "5f06696d-a54f-49c1-a798-53c871671160",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "HTTPError",
     "evalue": "500 Server Error: Internal Server Error for url: http://0.0.0.0:8000/v1/infer",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mHTTPError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[18], line 19\u001b[0m\n\u001b[1;32m      4\u001b[0m prompt \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mfirst person view from a camera in a car driving down a two lane neighborhood street, \u001b[39m\u001b[38;5;124m\"\u001b[39m \\\n\u001b[1;32m      5\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mviewed from the dashcam as we drive down the street. \u001b[39m\u001b[38;5;124m\"\u001b[39m \\\n\u001b[1;32m      6\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mThe camera faces forward. There are nice houses and sidewalks \u001b[39m\u001b[38;5;124m\"\u001b[39m \\\n\u001b[1;32m      7\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124min this suburban area with green grass front yards and flower gardens and large oak trees. \u001b[39m\u001b[38;5;124m\"\u001b[39m \\\n\u001b[1;32m      8\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mIt is a rainy day and there are grey clouds overhead. \u001b[39m\u001b[38;5;124m\"\u001b[39m \\\n\u001b[1;32m      9\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mThe road has puddles on it, reflecting the sky overhead. The windshield wipers flash by.\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m     11\u001b[0m response \u001b[38;5;241m=\u001b[39m requests\u001b[38;5;241m.\u001b[39mpost(\n\u001b[1;32m     12\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhttp://0.0.0.0:8000/v1/infer\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[1;32m     13\u001b[0m     json\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mdict\u001b[39m(prompt\u001b[38;5;241m=\u001b[39mprompt),\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     17\u001b[0m     }\n\u001b[1;32m     18\u001b[0m )\n\u001b[0;32m---> 19\u001b[0m \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraise_for_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     21\u001b[0m data \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mjson()\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mb64_video\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m     22\u001b[0m video_bytes \u001b[38;5;241m=\u001b[39m base64\u001b[38;5;241m.\u001b[39mb64decode(data)\n",
      "File \u001b[0;32m/usr/local/lib/python3.10/dist-packages/requests/models.py:1026\u001b[0m, in \u001b[0;36mResponse.raise_for_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1021\u001b[0m     http_error_msg \u001b[38;5;241m=\u001b[39m (\n\u001b[1;32m   1022\u001b[0m         \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mstatus_code\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m Server Error: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mreason\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m for url: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39murl\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m   1023\u001b[0m     )\n\u001b[1;32m   1025\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m http_error_msg:\n\u001b[0;32m-> 1026\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m HTTPError(http_error_msg, response\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m)\n",
      "\u001b[0;31mHTTPError\u001b[0m: 500 Server Error: Internal Server Error for url: http://0.0.0.0:8000/v1/infer"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "import base64\n",
    "\n",
    "prompt = \"first person view from a camera in a car driving down a two lane neighborhood street, \" \\\n",
    "    \"viewed from the dashcam as we drive down the street. \" \\\n",
    "    \"The camera faces forward. There are nice houses and sidewalks \" \\\n",
    "    \"in this suburban area with green grass front yards and flower gardens and large oak trees. \" \\\n",
    "    \"It is a rainy day and there are grey clouds overhead. \" \\\n",
    "    \"The road has puddles on it, reflecting the sky overhead. The windshield wipers flash by.\"\n",
    "\n",
    "response = requests.post(\n",
    "    \"http://0.0.0.0:8000/v1/infer\",\n",
    "    json=dict(prompt=prompt),\n",
    "    headers={\n",
    "        \"Accept\": \"application/json\",\n",
    "        \"Content-Type\": \"application/json\",\n",
    "    }\n",
    ")\n",
    "response.raise_for_status()\n",
    "\n",
    "data = response.json().get(\"b64_video\")\n",
    "video_bytes = base64.b64decode(data)\n",
    "\n",
    "with open(\"video.mp4\", \"wb\") as video_file:\n",
    "    video_file.write(video_bytes)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "aab73cf0-cb0f-434e-a7ff-71f204f121ca",
   "metadata": {},
   "source": [
    "## Stop the Container"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b34f54a0-334b-4033-9ede-5fc84dd72016",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES\n",
      "Error response from daemon: No such container: Llama3-8B-Instruct\n"
     ]
    }
   ],
   "source": [
    "!docker ps -a && \\\n",
    "docker stop cosmos-predict1-7b-text2world"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "044a6fb4-0c7d-453e-bd5c-52a66c65b1e9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
